src/histos.jl

```diff
@@ -3,6 +3,7 @@ using Statistics
 using StatsBase
 # histograms
 
+#Necessary? Inf is defined.
```

Comment:
Actually, you should use the standard `typemax` function, rather than `Inf`, which will ensure that you get the correct answer, whatever type you happen to be using.

Reply:
I found `typemax` when I was looking into how to improve those functions. The comment is from my first read-through and probably doesn't belong in this PR so I'll remove the commit before merging but a future PR will make the changes to get rid of `bigp` etc